source/extensions/filters/http/cache/simple_http_cache/simple_http_cache.cc

```diff
@@ -124,17 +132,22 @@ SimpleHttpCache::Entry SimpleHttpCache::lookup(const LookupRequest& request) {
   if (VaryHeader::hasVary(*iter->second.response_headers_)) {
     return varyLookup(request, iter->second.response_headers_);
   } else {
+    Http::ResponseTrailerMapPtr trailers_map =
```

Comment:
Nit: if you rewrite this as an if-statement, it will be the same number of lines, but might be easier to read.